assets/scripts/streets/StreetMetaWidth.jsx

```diff
@@ -76,17 +76,15 @@ export class StreetMetaWidth extends React.Component {
   }
 
   normalizeStreetWidth (width) {
+    const { unitSettings } = this.props
+
     if (width < MIN_CUSTOM_STREET_WIDTH) {
       width = MIN_CUSTOM_STREET_WIDTH
     } else if (width > MAX_CUSTOM_STREET_WIDTH) {
       width = MAX_CUSTOM_STREET_WIDTH
     }
 
-    var resolution = getSegmentWidthResolution()
-    if (!resolution) {
-      // TODO remove need to set a default here.
-      resolution = 0.25
-    }
+    const resolution = (unitSettings && unitSettings.resolution) || 0.25
```

Comment:
I just noticed something else, actually. This default value is probably not necessary anymore. This was a momentary hack for a situation where the resolution wouldn't be set, for some reason. Looking at the Redux store now, it should always have a resolution value except during the store's `initialState` (but we can set that too).